docs/security/trust/trust_key_mng.md

```diff
@@ -0,0 +1,63 @@
+
+Trust for an image tag is managed through the use of keys. Docker's content
+trust makes use four different keys: 
+
+| Key                 | Description                                                                                                                                                                                                                                                                                                                                                                         |
+|---------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| offline key            | Root of trust for a image tag. When trust is enabled, you create this key once. Then, you can use this key to create tagging keys.  |
+| target and snapshot | These two keys are known together as the repository pair.  When trust is enabled, you create this key when you add a new image repository.  If you have the offline key, you can export the tagging key and allow other images creators to sign image tags.       |
+| timestamp           | This key applies to a repository. It allows Docker repositories to have freshness security guarantees without requiring periodic content refreshes on the client's side.  Each tag is signed with a tagging key and a timestamp key.                                                                                                                |
+
+With the exception of timestamps, all the keys are generated and stored locally
+client-side. The timestamp is generated and stored in a Docker registry. 
+
+## Choosing a passphrase
+
+The passphrase you chose for both the offline key and your tagging key should be
+randomly generated and stored in a password manager. Having the tagging key allow
+users to sign image tags on a repository. 
+
+## Back up your keys
+
+All the Docker trust keys are stored encrypted using the passphrase you provide
+on creation. Even so, you should still take care of the location where you back them up.
+Good practice is to create two encrypted USB keys.
+
+It is very important that you backup your keys to a safe, secure location. Loss
+of the tagging key is recoverable; loss of the offline key is not.
+
+The Docker client stores the keys in the `~/.docker/trust/private` directory.
+Before backing them up, you should `tar` them into an archive:
+
+```bash
+$ tar -zcvf private_keys_backup.tar.gz ~/.docker/trust/private
+$ chmod 600 private_keys_backup.tar.gz
+```
+
+## Lost keys
+
+If a creator loses keys it means losing the ability to sign trusted content for
+your repositories.  If you lose a key, contact [Docker
+Support](https://support.docker.com) (support@docker.com) to reset the image tag
+state. 
+
+This loss also requires **manual intervention** from every consumer that pulled
+the tagged image prior to the loss. These users will see ??? when they attempt
```

Comment:
This is the error they will get: `one or more filesystem layers failed to download, verify, or extract successfully`. It's generic because it could come from a corrupted image, an actual malicious attempt or anything in between.